src/vs/workbench/browser/actions/quickAccessActions.ts

```diff
@@ -147,10 +151,33 @@ registerAction2(class QuickAccessAction extends Action2 {
 
 	run(accessor: ServicesAccessor, prefix: undefined): void {
 		const quickInputService = accessor.get(IQuickInputService);
-		quickInputService.quickAccess.show(typeof prefix === 'string' ? prefix : undefined, { preserveValue: typeof prefix === 'string' /* preserve as is if provided */ });
+		const searchText = getFileSearchText(accessor) ?? '';
+		quickInputService.quickAccess.show(typeof prefix === 'string' ? prefix : searchText, { preserveValue: typeof prefix === 'string' /* preserve as is if provided */ });
 	}
 });
 
+
+function getFileSearchText(accessor: ServicesAccessor): string | null {
```

Comment:
Instead of returning null, this should just return empty string since above, you're just doing `?? ''` above

Reply:
Done!